mne/io/brainvision/brainvision.py

```diff
@@ -222,49 +250,25 @@ def _read_segments_c(raw, data, idx, fi, start, stop, cals, mult):
         _mult_cal_one(data, block, idx, cals, mult)
 
 
-def _read_vmrk_events(fname, event_id=None, trig_shift_by_type=None):
-    """Read events from a vmrk file.
+def _read_vmrk(fname):
```

Comment:
I'm all in favor of splitting functions (and have them all with 5 lines or less), but maybe instead of returning all the elements of annotations we could just directly spit `Annotattions`. (we miss **sfreq**, though)